tests/test_pano_render_descriptors.py

```
@@ -0,0 +1,118 @@
+import json
+import subprocess
+import textwrap
+import unittest
+from pathlib import Path
+
+
+REPO_ROOT = Path(__file__).resolve().parent.parent
+
+
+def run_node_json(script: str):
+    proc = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)